src/Serilog.Sinks.Splunk/Sinks/Splunk/EventCollectorRequest.cs

```diff
@@ -32,6 +32,11 @@ internal SplunkEvent(string logEvent, string source, string sourceType, string h
                 jsonPayLoad = jsonPayLoad + @",""index"":""" + index + @"""";
```

Comment:
Also we have this, could be improved however we do batch if needed.  Needs to be revisited with the `epoch` time allocated to each event.

https://github.com/serilog/serilog-sinks-splunk/blob/dev/src/Serilog.Sinks.Splunk/Sinks/Splunk/EventCollectorSink.cs#L183